xcode/Subconscious/Shared/UIKit/UIHostingView.swift

```diff
@@ -17,27 +17,19 @@ class UIHostingView<Content: View>: UIView {
     /// A reference to the hosting controller for this UIHostingView.
     /// We use this to remove hostingController from the parent controller
     /// on deinit.
-    private var hostingController: UIHostingController<Content>
+    private var hostingController = UIHostingController<Content?>(
```

Comment:
Update approach to allow for an optional SwiftUI view. This lets us initialize the view without having to immediately pass in a SwiftUI view.

We do this because `.collectionView.dequeueReusableCell(withReuseIdentifier:for:)` is not tied to a typical initialization approach. Instead, it is initialized with zero arguments, and then configured. So, we initialize with no SwiftUI view, and then configure via `update(parentController:rootView:)`.

Reply:
Could document the above as a code comment.

Reply:
Done